src/test/java/com/devcomanda/easymeetup/controller/MeetupControllerTest.java

```diff
@@ -0,0 +1,47 @@
+package com.devcomanda.easymeetup.controller;
+
+import com.devcomanda.easymeetup.entity.Meetup;
+import com.devcomanda.easymeetup.repository.MeetupRepository;
+import com.devcomanda.easymeetup.service.MeetupService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(MeetupController.class)
```

Comment:
If you use this annotation and approach, you should use the mockmvc for testing your controller,
 You can read more here  - https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-testing.html#boot-features-testing-spring-boot-applications-testing-autoconfigured-mvc-tests

Reply:
Please rewrite this tests, if you have questions, you can ask me in a telegram